packages/theme/src/react/with-theme.js

```diff
@@ -12,17 +12,29 @@ const getDisplayName = Component => {
   return Component.displayName || Component.name || 'Component'
 }
 
-export default function withTheme(BaseComponent) {
+export default function withTheme(BaseComponent, options = {}) {
+  const { forwardRef = true } = options
```

Comment:
i added an escape hatch if we don't want to use the ref forwarding. this is similar in other hocs i've seen(redux#connect) but it's not actually used anywhere at the moment.

Reply:
Ok, cool.  In what case would one want to avoid ref forwarding?

Reply:
i don't actually know. maybe i should just remove this and we can add in later if we need to. i'm now thinking it was a react 15 compat thing....